reward/doc.go

```diff
@@ -14,5 +14,62 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package reward implements the Klaytn reward system.
+/*
+Reward Package
+Package Reward implements the Klaytn Reward System.
+Klaytn Reward System manages stakingInfo and distributes block reward.
+
+Managing stakingInfo
+Klaytn uses weightedRandom policy to choose a block proposer.
+It means, the percentage of becoming a block proposer depends on how much klay a node has staked.
+Therefore, a node staked more than others will have more opportunities than other nodes.
+
+StakingInfo is a data including stakingAmount and addresses(node, staking, reward, poc, kir).
```

Comment:
(node, staking, reward, poc, kir) --> (node, staking, reward, poc and kir)

Reply:
reflected